src/System.Private.Uri/System.Private.Uri.sln

```diff
@@ -1,8 +1,8 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
```

Comment:
Should these changes be included?

Reply:
No. Visual Studio tooling creates noise in our PRs unfortunately.

Reply:
Reverted.

Reply:
this isn't reverted?

Reply:
On second though, maybe this change should be made? AFAIK most of us are using VS19 at the moment.

Reply:
We are still supporting VS2017. When we migrate to the new repository we will do the cut. Definitely not now.

Reply:
Yes, all the changes in the file should be reverted.

Reply:
I reverted them once, but then VS changed it again and I accidentally committed the changed version one more time.